packages/compile-solidity/test/test_supplier.js

```diff
@@ -170,10 +167,23 @@ describe("CompilerSupplier", function () {
         options: cachedOptions
       });
 
-      assert(await fse.exists(expectedCache), "Should have cached compiler");
+      const cachedCompilerFilenames = fse.readdirSync(compilerCacheDirectory);
+      const compilerFilename = cachedCompilerFilenames.find(filename => {
+        return filename.includes("v0.4.21+commit.dfe3193c");
```

Comment:
This is truly nitpicking and you don't need to change this, but `filename => { return foo; }` could instead just be `filename => foo`.